compact_test.go

```diff
@@ -604,27 +672,40 @@ func TestCompaction_populateBlock(t *testing.T) {
 				{
 					{
 						// Includes chunk partially outside.
-						lset:   map[string]string{"a": "b"},
-						chunks: [][]sample{{{t: 1}, {t: 2}}, {{t: 10}, {t: 20}, {t: 21}, {t: 30}}},
+						lset: map[string]string{"a": "b"},
+						chunks: [][]sample{
+							{{t: 1}, {t: 2}},
+							{{t: 10}, {t: 20}, {t: 21}, {t: 30}},
+						},
 					},
 					{
 						// Includes chunk completely outside.
 						// Regression test against https://github.com/prometheus/tsdb/issues/347.
-						lset:   map[string]string{"a": "c"},
-						chunks: [][]sample{{{t: 1}, {t: 2}}, {{t: 10}, {t: 20}}, {{t: 21}, {t: 30}}},
+						lset: map[string]string{"a": "c"},
+						chunks: [][]sample{
+							{{t: 1}, {t: 2}},
+							{{t: 10}, {t: 20}},
+							{{t: 21}, {t: 30}},
```

Comment:
first I was confused why is this test different than the one before and after adding the new lines it is clear now so I added new lined for all test.